application/helpers/general_functions_helper.php

```diff
@@ -116,7 +116,7 @@ function alphabet()
 }
 
 
-function full_languages_dropdown($id='', $selected_id=''  )
+function full_languages_dropdown($id='', $selected_id='', $extra_search_options=false)
```

Comment:
It's not really `$extra_search_options` though, it's more `$show_all_but_english` - maybe you have plans for this argument in the future, but for now I'd rather we be very explicit about what it's for. We can always change it later.